### ISSUE TITLE
Bug: `--test=PATH` option doesn't work with mocha.

### ISSUE THREAD

**Issue by ryan-roemer** [TestArmada/magellan#74]
Reproduction

* Download https://github.com/walmartlabs/little-loader and `npm install` and `npm run install-dev`
* Try to filter with `magellan --tests=./test/func/spec/advanced.spec.js` (or any variation to get to just that test)
* **Expected Behavior**: Runs only that test.
* **Actual Behavior**: Errors out with:

```
$ magellan --test=test/func/spec/advanced.spec.js 
Loaded magellan configuration from:  /Users/rye/scm/wm/little-loader/magellan.json
Sauce configuration OK
Using test filter:  test/func/spec/advanced.spec.js
Error: no tests found
```

**Comment by ryan-roemer** [comment]
Note: Separate magellan config is a workaround for now. See., e.g. https://github.com/walmartlabs/little-loader/pull/7/files

**Comment by Maciek416** [comment]
Confirmed

**Comment by Maciek416** [comment]
Fixed. Note that the fix requires installing magellan `5.0.0` or greater, **and** installing the mocha plugin like so:

```shell
npm install --save-dev testarmada-magellan-mocha-plugin
```

in `magellan.json`, **framework** being set to `rowdy-mocha` is fine for now, but for future-proofing, I recommend moving to:
```json
{
  "framework": "testarmada-magellan-mocha-plugin"
}
```

The former will eventually be deprecated with a warning.
